src/openforms/payments/contrib/ogone/tests/test_plugin.py

```diff
@@ -86,6 +86,7 @@ def test_payment(self):
             "STATUS": OgoneStatus.payment_requested,
             "GIROPAY_ACCOUNT_NUMBER": "1",  # hashed but not interesting
             "UNKNOWN_PARAM": "1",  # not hashed
+            "PAYID": "4249708957",
```

Comment:
hrm, any idea if this was not originally included in the test by design or just some oversight?

It kind of implies that the payment plugin doesn't properly validate/check the claims of the Ogone documentation about the guaranteed/required parameters :thinking:

Reply:
I'm not sure, since we didn't do anything with the `PAYID` I think it might be an oversight. I would assume that if the payment is successful, we should get a PAYID?

Reply:
yeah, I also think we'll get it if the payment via Ogone failed for whatever reason (e.g. iDeal but not enough money in your account -> bank will fail payment -> iDeal will fail payment -> Ogone will fail the payment), but there will *probably* be a `PAYID` generated in Ogone. I have no idea if we'll get that information or not

Reply:
I've changed it to assume we always get a PAYID and it seems to work if I try a failure flow with Ogone